src/explore-education-statistics-frontend/src/services/apiDataSetService.ts

```diff
@@ -33,7 +33,7 @@ export interface ApiDataSetVersion {
   status: ApiDataSetStatus;
   published: string;
   withdrawn?: Date;
-  notes: string;
+  notes?: string;
```

Comment:
This isn't optional according to the backend view models? If there aren't any notes, this will just be an empty string.

Think this should be reverted?